src/components/NavigationPopupMenu/NavigationPopupMenu.js

```diff
@@ -17,6 +17,17 @@ import useLockBodyScroll from './../../utils/useLockBodyScroll'
 
 import LoaderSpinner from './../LoaderSpinner'
 
+/**
+ * Component that renders the NavigationPopupMenu component.
+ *
+ * @component
+ *
+ * @param {} loggedIn - Parameter is not used within this component.
+ * @param {booelan} showBanner - Parameter that if set to true, will show the banner.
+ * @param {} logout - Parameter is not used within this component.
```

Comment:
Good catch, feel free to remove the parameter and this doc line to clean it up :)

Reply:
Thanks, will remove the parameters from now on.